commit 8bfaa898b32370bbfff9bf37fc833d7abc4b8b67
Author: Bryan Drewery <bryan@shatow.net>
Date:   Sat Aug 28 11:57:56 2021 -0700

    distclean: Fix reading make.conf
    
    Fixes #910
    Fixes fa0308152e6 ("Move make.conf ports_env handling to its own not-spammy file")
    
    (cherry picked from commit 95971acb6ffd51b2432a38efd1111dd189f68edd)

diff --git src/share/poudriere/distclean.sh src/share/poudriere/distclean.sh
index 08889f1dd..f77163fc4 100755
--- src/share/poudriere/distclean.sh
+++ src/share/poudriere/distclean.sh
@@ -144,7 +144,7 @@ for PTNAME in ${PTNAMES}; do
 
 	__MAKE_CONF=$(mktemp -t poudriere-make.conf)
 	export __MAKE_CONF
-	setup_ports_env "/" "${__MAKE_CONF}"
+	setup_ports_env "" "${__MAKE_CONF}"
 	if [ -z "${NO_PACKAGE_BUILDING}" ]; then
 		echo "BATCH=yes"
 		echo "PACKAGE_BUILDING=yes"
